content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -1,36 +1,31 @@
 ---
 reviewers:
 - janetkuo
-title: TTL Controller for Finished Resources
+title: Automatic Clean-up for Finished Jobs 
 content_type: concept
 weight: 70
 ---
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
 
-The TTL controller provides a TTL (time to live) mechanism to limit the lifetime of resource
-objects that have finished execution. TTL controller only handles
-{{< glossary_tooltip text="Jobs" term_id="job" >}} for now,
-and may be expanded to handle other resources that will finish execution,
-such as Pods and custom resources.
-
-This feature is currently beta and enabled by default, and can be disabled via 
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-`TTLAfterFinished` in both kube-apiserver and kube-controller-manager.
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} provides a 
+TTL (time to live) mechanism to limit the lifetime of resource objects that 
+have finished execution. TTL controller only handles 
+{{< glossary_tooltip text="Jobs" term_id="job" >}}.
 
 <!-- body -->
 
-## TTL Controller
+## TTL-after-finished Controller
 
-The TTL controller only supports Jobs for now. A cluster operator can use this feature to clean
+The TTL-after-finished controller is only supported for Jobs. A cluster operator can use this feature to clean
```

Comment:
It we change this, we should also change other references of "resources" to "Jobs".

Reply:
Great comment @janetkuo. I have updated the references of `resources` to `jobs`.